doc/whats_new/v1.4.rst

```diff
@@ -354,6 +354,15 @@ Changelog
   dtype as categories in the algorithm. `categorical_features="from_dtype"` will become
   the default in v1.6. :pr:`26411` by `Thomas Fan`_.
 
+- |Feature| the `categorical_features="from_dtype"` option of
```

Comment:
I would say this is a major feature. I am wondering if we should merge it with the previous entry to simplify the message (it works for both Polars and Pandas).